ui/src/views/ZaaktypeEditView.tsx

```diff
@@ -1,177 +1,180 @@
+      const rows = e?.rawMessage as any;
+      const messages = rows
+        .map((errorPairs: Record<string, string>, rowIndex: number) =>
+          Object.entries(errorPairs).map(
+            ([key, val]) => `${ucFirst(key)} (${rowIndex + 1}): ${val}`
+          )
+        )
+        .flatMap((e: string[]) => e);
+      setErrors(messages);
     }
-  };
 
-  const handleQuit = () => {
-    const sameData = true;
-
-    const navigateQuit = () => {
-      closeSnackbar();
-      navigate('..', { relative: 'path' });
-    };
-
-    if (!sameData) {
-      enqueueSnackbar('Er zijn niet opgeslagen wijzigingen', {
-        preventDuplicate: true,
-        variant: 'unsavedChanges',
-        hint: 'Weet je zeker dat je wilt annuleren?',
-        persist: true,
-        action: () => (
-          <Box sx={{ p: 2 }}>
-            <Typography gutterBottom variant="body2">
-              Zaaktype toch opslaan?
-            </Typography>
-            <Button size="small" color="primary" startIcon={<Close />} onClick={navigateQuit}>
-              Ja, ik wil annuleren
-            </Button>
-          </Box>
-        ),
-        anchorOrigin: {
-          vertical: 'top',
-          horizontal: 'right',
-        },
-      });
-    } else {
-      navigateQuit();
-    }
+    setLoading(false);
   };
 
   return (
-    <Stack direction={'row'} flexWrap={'wrap'} width={'100%'} spacing={spacings.xlarge} useFlexGap>
-      <Stack
-        direction={'row'}
-        justifyContent={'space-between'}
-        alignItems={'center'}
-        flexWrap={'wrap'}
-        width={'100%'}
-        spacing={spacings.large}
-        useFlexGap
-      >
-        <Typography variant="h1">
-          {loadingState ? <Skeleton variant="text" width={200} /> : value?.zaaktype?.omschrijving}
-        </Typography>
-        <Stack direction={'row'} spacing={spacings.small}>
-          <Button variant="outlined" onClick={handleQuit}>
-            Annuleren
-          </Button>
-          <Button variant="contained" onClick={handleSave} disableElevation>
-            Opslaan
-          </Button>
-        </Stack>
-        <Stack spacing={spacings.small} width="100%">
-          <Typography variant="h2">
-            {loadingState ? (
-              <Skeleton variant="text" width={320} />
-            ) : (
-              'Koppel informatieobjecttypen'
-            )}
-          </Typography>
-          <Typography variant="body2">
-            {loadingState ? (
-              <Skeleton variant="text" width={340} />
-            ) : (
-              'Hier kunt u de gekoppelde informatieobjecttypen wijzigen.'
-            )}
-          </Typography>
-          <BulkEditor
-            apiRef={apiRef}
-            loading={loadingState}
-            zaaktype={value?.zaaktype!}
-            informatieobjecttypen={value?.informatieobjecttypen!}
-          />
-        </Stack>
-      </Stack>
-    </Stack>
+    <List
+      errors={errors}
+      fields={[
+        'omschrijving',
```

Comment:
If I click on the link icon, then it gives me the possibility to update the description. I don't think this is intended.
![Screenshot from 2024-04-09 16-56-54](https://github.com/maykinmedia/open-zaaktypebeheer/assets/19154114/7781ab9b-333e-4e0c-96f5-da3c6b70b1b2)